dns_sd_bonjour.c

```diff
@@ -44,124 +52,197 @@ static void __cfnet_browser_cb (
 	CFStreamError* 			error,
 	void* 					info)
 {
-	CFStreamError anError;
-	
+	CFStreamError	anError;
+
 	if ((flags & kCFNetServiceFlagIsDomain) != 0) {
-		ERROR("DNS SD: FATAL. Callback called for domain, not service.\n");
+		ERROR("DNS SD: FATAL! Callback called for domain, not service.\n");
 		goto stop_browsing;
 	}
 	
-	struct cfnet_discovery_data *dd = (struct cfnet_discovery_data *)info;
+	struct dns_sd_discovery_data *dd = (struct dns_sd_discovery_data *)info;
```

Comment:
this isn't new to the file;
but these declarations in the middle of the code-block are C99 specific;
do we want to maintain/have C90 compatibility?

Reply:
I think we have lots of C99 isms in the codebase already - don't we?

Reply:
yep; don't mind c99 or c90 or c11;
let's hope nobody asks us for anything old